tests/installation/disable_grub_timeout.pm

```diff
@@ -78,6 +78,11 @@ sub run {
     wait_still_screen(1);
     # ncurses uses blocking modal dialog, so press return is needed
     send_key 'ret' if check_var('VIDEOMODE', 'text');
+    # make sure sure value is -1 not 0
+    if (check_var('VIDEOMODE', 'text') && $timeout eq '-1') {
+        wait_still_screen(1);
+        send_key 'down';
+    }
```

Comment:
Can we add a needle check here to make sure it has be set to "-1"?

Reply:
I don't want to add needle, just because IMO it's not needed.
It works, but sometimes "1" get somehow lost, as workaround pressing down will make sure the timeout is "-1".
But if by any chance this will happen again I will add needle. :wink: